README.md

```diff
@@ -100,6 +100,12 @@ one package, you can run:
 yarn workspace <package_name> test
 ```
 
+If you are testing the ExpiringMultiPartyClient, you will need to set an environment variable so that the test will work with Ganache.
+
+```
+export FROM_BLOCK=0
```

Comment:
This is a bit annoying to ask of devs but alas, I wish there was a way to start ganache at a certain block

Reply:
same :\